server/code/setup/cpdb

```diff
@@ -236,8 +260,12 @@ if __name__ == "__main__":
         print("ERROR: --drop can not be used with --schema-only")
         sys.exit(1)
 
+    if options.ssl and options.sslfactory and options.sslcertpath:
+            print("ERROR: ssl, sslfactory, and sslcertpath is not supported together.")
```

Comment:
Grammar: "...and sslcertpath *are* not supported together"

Also, indentation looks wrong for this level compared to the rest of the document.

Reply:
Updated as suggested.